Source/Core/VideoCommon/FramebufferShaderGen.cpp

```diff
@@ -605,6 +613,9 @@ std::string GenerateTextureReinterpretShader(TextureFormat from_format, TextureF
           "  }\n";
   }
   break;
+
+  default:
+    return "{}\n";
```

Comment:
Could add please add a `WARN_LOG(VIDEO, "From format %u not supported", static_cast<u32>(from_format));` here in case this does get executed?